docs/configuration/networking.md

```diff
@@ -0,0 +1,133 @@
+---
+id: networking
```

Comment:
this was originally the index page to the Networking Section. Ideally, this file would be index to the 'Internet Access' section, but we'd have to rewrite the intro to it to talk about it

Reply:
There is no index pages here, per se, here, afaik.

The best we can do is make an "Introduction" page and set it as position one, but that's about it.

Reply:
There's also no Networking section anymore.